integration_tests/althea_proto/src/lib.rs

```diff
@@ -15,3 +15,72 @@ pub mod microtx {
         include!("prost/microtx.v1.rs");
     }
 }
+
```

Comment:
Here I compiled all of the canto and ethermint protos, which requires that we copy those proto definitions into the third_party folder. I minimized the imports of their proto dependencies and modified the linter configs to stop producing erros we cannot resolve.